repo/packages/P/percona-mongo/1/package.json

```diff
@@ -0,0 +1,25 @@
+{
+    "packagingVersion": "4.0",
+    "upgradesFrom": [
+        "*"
+    ],
+    "downgradesTo": [
+        "*"
+    ],
+    "minDcosReleaseVersion": "1.10",
+    "name": "percona-mongo",
+    "version": "0.2.0-3.4.13",
+    "maintainer": "mesosphere@percona.com",
+    "description": "Percona Server for MongoDB Replica Set on Mesosphere DC/OS",
+    "selected": false,
+    "framework": true,
+    "tags": [
+        "mongo",
+        "mongodb",
+        "percona",
+        "nosql"
+    ],
+    "postInstallNotes": "DC/OS Mongo is being installed!\n\n\tDocumentation: https://www.percona.com/software/mongo-database/documentation and https://docs.mongodb.com/manual\n\tMongoDB Release Notes: https://docs.mongodb.com/manual/release-notes",
```

Comment:
Change `postInstallNotes` to: `The DC/OS Mongo service is being installed. \n\n\tDocumentation: https://docs.mesosphere.com/service-docs/percona-mongo/\n\tIssues: mesosphere@percona.com`